### PR TITLE
Improve ip address parsing in rules

### PR DESCRIPTION
**Pull request by shaleman**
minor fix to CIDR parsing

### CODE REVIEW

File: ofnetPolicy.go [hunk 1]
```diff
@@ -200,28 +201,53 @@ func (self *PolicyAgent) AddRule(rule *OfnetPolicyRule, ret *bool) error {
...
-		ipDaMask = &ipMask
 	}
 
 	// parse src ip
 	if rule.SrcIpAddr != "" {
-		ipSav, ipNet, err := net.ParseCIDR(rule.SrcIpAddr)
-		if err != nil {
-			log.Errorf("Error parsing src ip %s. Err: %v", rule.SrcIpAddr, err)
-			return err
+		if strings.Contains(rule.SrcIpAddr, "/") {
```
Comment (jojimt) [inline ofnetPolicy.go]: Nit: probably should move this to a func, because it is doing the exact same thing.

Comment (shaleman) [inline ofnetPolicy.go]: Sure. 
Coming to think of it, I think there is a ParseCIDR function that @abhinandanpb added. Let me see if i can reuse it.

**Comment by shaleman** [comment]
@jojimt fixed it. PTAL

**Comment by jojimt** [comment]
LGTM

**Pull request merged**